config/components/webhook/manifests.yaml

```diff
@@ -24,6 +24,26 @@ webhooks:
     resources:
     - jobsets
   sideEffects: None
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: system
+      path: /mutate-v1-pod
+  failurePolicy: Fail
```

Comment:
I am concerned that this will block the whole cluster if jobset-controller is not functioning well. Is there a way to setup the webhook so only specific pods are forwarded? a selector somewhere?

Reply:
it seems there is: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-objectselector

so we need to set this label selector on pods created (indirectly) via JobSet

Reply:
I added an event filter for the pods the controller watches but don't see a similar way to do this for the webhook:

<img width="786" alt="image" src="https://github.com/kubernetes-sigs/jobset/assets/105610547/802b5d17-2611-45d4-be81-289256a48437">


I looked at the doc you linked but am still figuring out how to add the objectSelector to Go code

Reply:
we can change this manually here